packages/aws-rfdk/bin/stage-deadline.ts

```diff
@@ -49,6 +55,22 @@ while (n < args.length) {
   n++;
 }
 
+// Automatically populate the installer & recipe URI using the version, if it is provided.
+if (deadlineRelease !== '') {
+  if (!/^10\.\d+\.\d+\.\d+$/.test(deadlineRelease)) {
+    usage(1);
+  }
+
+  // installer URI only if not provided as an arg
+  if (deadlineInstallerURI === '') {
```

Comment:
Here, both `deadlineInstallerURI` and `dockerRecipesURI` values are taking precedence over deadlineVersion. 
Shouldn't it be otherwise? If not, we should update the usage accordingly for the precedence if both values are provided.

Reply:
I believe the intention here is that you can provide `deadlineReleaseVersion`, but then also provide either of the URI's as well so that the URI's that that would get generated by using `deadlineReleaseVersion` can be overridden.

I do think that this adds some unnecessary complexity to understanding its usage. If we don't see a realistic use case where the user needs to override just one of these URI's, can we just make the usage either `stage-deadline --deadlineReleaseVersion <version>` or `stage-deadline --deadlineInstallerURI <uri> --dockerRecipesURI <uri>` with no combinations?

Reply:
It's actually what I was going for. I could be clearer in the comments & command documentation, though. My thinking is that you can provide the version, but still use the other options to override the default URIs.

Reply:
I think that since Yash and I both stumbled on why it was like this and it's so unlikely that anyone other than RFDK devs would need to do it, that it would be better to remove this as an option. If you really do want to override one but not the other, it's simple enough to provide both URI's.